pycbc/events/single.py

```diff
@@ -153,7 +153,7 @@ def check(self, trigs, data_reader):
 
     def calculate_ifar(self, newsnr, duration):
         if self.fit_info['fixed_ifar']:
-            return self.fit_info['fixed_ifar']
+            return self.fit_info['fixed_ifar'][self.ifo]
```

Comment:
@GarethDaviesGW should probably comment on these changes to `pycbc/events/single.py`.

Reply:
@GarethDaviesGW This change is needed, as the code is currently non-functional for setting a fixed ifar. I think when the if statement at the top of the code was added, it was forgotten that this is dict that is keyed by the ifo.

Reply:
Yes, sorry I must have missed that before. I think when moving code into the new structure I missed that it was set in from_cli before. I'm happy for it to be here instead